test/threadsafe_function/threadsafe_function_existing_tsfn.js

```diff
@@ -12,8 +12,8 @@ module.exports = Promise.all([
 async function test(binding) {
   const testCall = binding.threadsafe_function_existing_tsfn.testCall;
   
-  assert(typeof await testCall({ blocking: true,  data: true  }) === "number");
-  assert(typeof await testCall({ blocking: true,  data: false }) === "undefined");
-  assert(typeof await testCall({ blocking: false, data: true  }) === "number");
-  assert(typeof await testCall({ blocking: false, data: false }) === "undefined");
+  assert.strictEqual(typeof await testCall({ blocking: true,  data: true  }) , "number");
+  assert.strictEqual(typeof await testCall({ blocking: true,  data: false }) , "undefined");
+  assert.strictEqual(typeof await testCall({ blocking: false, data: true  }) , "number");
+  assert.strictEqual(typeof await testCall({ blocking: false, data: false }) , "undefined");
```

Comment:
@Nishikoh could you remove the space before the comma ","?
Es.
```js
assert.strictEqual(typeof await testCall({ blocking: false, data: false }), "undefined");
```

Reply:
Thank you for the review. Fixed it.